pkg/restapi/issuer/operation/models.go

```diff
@@ -27,6 +28,8 @@ type ProfileDataRequest struct {
 	OIDCClientParams            *OIDCClientParams `json:"oidcParams,omitempty"`
 	CredentialScopes            []string          `json:"scopes,omitempty"`
 	LinkedWalletURL             string            `json:"linkedWallet,omitempty"`
+	IssuerID                    string            `json:"issuerID,omitempty"`
```

Comment:
follow up: please add comments explaining usage of these params so that it can show up in documentation.